deployment/flexible_api_server.py

```
@@ -0,0 +1,564 @@
+    if detector is None:
+        return jsonify({
+            'status': 'unhealthy',
+            'error': 'Detector not initialized'
+        }), 503
+
+    status = detector.get_status()
+    status['status'] = 'healthy'
+    return jsonify(status)
+
+@app.route('/predict', methods=['POST'])
+def predict_emotion():
```

Ruff (0.12.2)

Missing return type annotation for public function `predict_emotion`

(ANN201)